tests/conftest.py

```diff
@@ -0,0 +1,94 @@
+import shutil
+import os
+import stat
+
+from pytest import fixture
+from pathlib import Path
+from subprocess import run
+from typing import Iterable
+from bodywork.constants import (
+    SSH_DIR_NAME,
+    GIT_SSH_COMMAND,
+)
+
+
+@fixture(scope="function")
+def test_namespace() -> str:
```

Comment:
Do we use this anywhere?

Reply:
Yes, it's used heavily in test_k8s_cluster.py.